lib/OpenQA/Utils.pm

```diff
@@ -166,6 +168,23 @@ sub testcasedir {
     return $dir;
 }
 
+sub gitrepodir {
+    my %args = (distri => '', version => '', repo => 'tests', @_,);
```

Comment:
this is a bit unusual syntax. What do you want to achieve? supply default values for variables if not supplied from outside? Why should repo default to "tests"?